templates/notebook_instance/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,5 @@
+if err = rm.requeueUntilCanModify(ctx, r); err != nil {
+	return r, err
+}
+
+rm.customPreDelete(r)
```

Comment:
I imagine the the stop operation isnt instantaneous. It takes some time from stopping -> stopped. Dont you need to requeue until then?

InService -> user calls delete -> controller calls StopNotebook -> Stopping -> requeue -> Stopped -> Delete